nautobot/docs/release-notes/version-2.0.md

```diff
@@ -138,6 +150,18 @@ Check out the [Region and Site Related Data Model Migration Guide](../user-guide
 
 `nautobot.utilities` no longer exists as a separate Python module or Django app. Its functionality has been collapsed into the `nautobot.core` app. See details at [Python Code Location Changes](../user-guide/administration/upgrading/from-v1/upgrading-from-nautobot-v1.md#python-code-location-changes).
 
+#### CSV Import/Export Re-implementations ([#254](https://github.com/nautobot/nautobot/issues/254))
+
+<!-- TODO: Probably not gonna be any impact on the user end but should be worth mentioning for developers. -->
```

Comment:
This absolutely impacts the user - the CSVs are now round-trip capable but have different structural expectations than either the "export" or "import" CSV formats in 1.x.